### PR TITLE
fix(solution common question): fix error handling when unauthorized subscription

### PR DESCRIPTION
**Pull request by dilin-MS**
Fix [bug](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10719249)

### CODE REVIEW

File: packages/fx-core/src/plugins/solution/fx-solution/commonQuestions.ts [hunk 1]
```diff
@@ -321,19 +321,32 @@ async function askCommonQuestions(
...
-        location: resourceGroupLocationFromProfile,
-      };
-    } else {
-      resourceGroupInfo = {
-        createNewResourceGroup: true,
-        name: resourceGroupNameFromProfile,
-        location: resourceGroupLocationFromProfile,
-      };
+    try {
+      const checkRes = await rmClient.resourceGroups.checkExistence(resourceGroupNameFromProfile);
```
Comment (1yefuwang1) [inline packages/fx-core/src/plugins/solution/fx-solution/commonQuestions.ts:325]: There is another checkExistence() call in this file. Is it deliberately left unchanged?

File: packages/vscode-extension/package-lock.json [hunk 2]
```diff
@@ -4,55 +4,6 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
```
Comment (1yefuwang1) [inline packages/vscode-extension/package-lock.json:6]: Why is package-lock.json changed?

File: packages/fx-core/src/plugins/solution/fx-solution/commonQuestions.ts (hunk 1 above)
Comment (dilin-MS) [reply packages/fx-core/src/plugins/solution/fx-solution/commonQuestions.ts:325]: I failed to find it. Could you please provide the code line? Thx

File: packages/vscode-extension/package-lock.json (hunk 2 above)
Comment (dilin-MS) [reply packages/vscode-extension/package-lock.json:6]: After updating the base branch `dev` and run `npm run setup`, this change is auto-done. Are we supposed to add the changes?

Comment (dilin-MS) [reply packages/vscode-extension/package-lock.json]: Removed the changes.

**Review by 1yefuwang1 — APPROVED** [review]

**Pull request merged**